.github/workflows/tests.yaml

```diff
@@ -1,94 +0,0 @@
-name: Tests
-on:
-  pull_request:
-    branches:
-      - main
-jobs:
-  docker-tests:
-    runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        dockerfile: [ "Dockerfile", "Dockerfile.alpine" ]
-    steps:
-      - uses: actions/checkout@v3
-
-      - name: Set up Docker Buildx
-        id: buildx
-        uses: docker/setup-buildx-action@master
-        with:
-          install: true
-
-      - name: Build test image
-        uses: docker/build-push-action@v3
-        with:
-          context: ./
-          file: ${{ matrix.dockerfile }}
-          push: false
-          pull: true
-          target: builder
-          tags: frankenphp:${{ github.sha }}-builder
-          builder: ${{ steps.buildx.outputs.name }}
-          cache-from: type=gha
-          cache-to: type=gha,mode=max
-          outputs: type=docker,dest=/tmp/.builder.tar
-
-      - name: Run tests
-        run: |
-          docker load -i /tmp/.builder.tar
-          docker run --rm frankenphp:${{ github.sha }}-builder "go test -a -v"
```

Comment:
We need to continue running the tests somewhere in the CI.

Reply:
Yes, this is just that I need help with this one.